docs/src/modules/javascript/pages/index.adoc

```diff
@@ -5,11 +5,11 @@ include::partial$attributes.adoc[]
 
 The Akka Serverless JavaScript SDK offers an idiomatic JavaScript language SDK for writing components. This page describes prerequisites for JavaScript development and basic requirements for a development project.
 
-NOTE: Lightbend provides Tier 1 support for the JavaScript SDK. See https://developer.lightbend.com/docs/akka-serverless/developing/sdk-support.html[an explanation of support tiers] for more information.
+NOTE: Lightbend provides Tier 1 support for the JavaScript SDK. See https://developer.lightbend.com/docs/akka-serverless/faq/index.html#_what_languages_are_supported[an explanation of support tiers] for more information.
 
 Your development project needs to include the Akka Serverless JavaScript SDK and logic to start the gRPC server. You define your components in gRPC descriptors and use `protoc` to compile them. Finally, you implement business logic for service components.
 
-To save the work of starting from scratch, the JavaScript xref:kickstart.adoc[code generation tool] creates a project, complete with descriptors and implementations. Or, you can start from one of our https://developer.lightbend.com/docs/akka-serverless/tutorial/index.html[example applications].
+To save the work of starting from scratch, the JavaScript xref:kickstart.adoc[code generation tool] creates a project, complete with descriptors and implementations. Or, you can start from one of our https://developer.lightbend.com/docs/akka-serverless/quickstart/prebuilt-samples.html[example applications].
```

Comment:
@retgits subjective decision here, could you please confirm?

Reply:
I'd probably link to the quickstarts in this case `https://developer.lightbend.com/docs/akka-serverless/quickstart/index.html`